Umbraco-Cloud/Set-Up/Manage-Security/index.md

```diff
@@ -0,0 +1,53 @@
+---
+versionFrom: 7.0.0
+---
+
+# Managing Transport Security
+
+Once you have added your own hostnames to your Umbraco Cloud project its possible to configure certain transport security options for all or for specific hostnames within your project.
+These security options all relate to the traffic that goes through your hostname from the origin (Umbraco Cloud) to the end user - meaning the protocols and encryption used to transport your website and assets from the webserver to the browser.
+
+The options that are currently available are
+- HTTP/2 (default: on)
+- TLS 1.3 (default: off)
+- Minimum TLS Version (default: 1.2)
+
+When a new hostname is added to a Project it will have the default settings applied. But you can change the defaults for your Project, so new hostnames will get the default settings to you have chosen.
```

Comment:
```suggestion
When a new hostname is added to a Project it will have the default settings applied. But you can change the defaults for your Project, so new hostnames will get the default settings you have chosen.
```